### ISSUE TITLE
Retain the imported status of pilots.

### ISSUE THREAD

**Issue by philderbeast** [FAI-CIVL/FAI-Airscore#245]
When I import the 2020 Meduno `*.fsdb`, keeping it external and publishing, I see pilots that are ABS in task 1.

![Screen Shot 2021-04-15 at 1 05 28 PM](https://user-images.githubusercontent.com/633283/114909971-5393c280-9deb-11eb-8b8a-ef3ec3333c3a.png)

When I convert to internal, import tracks and score these same pilots are now NYP. Could we retain the status of pilots when converting to internal please, at least for pilots without track logs?

![Screen Shot 2021-04-15 at 1 05 45 PM](https://user-images.githubusercontent.com/633283/114909970-5393c280-9deb-11eb-9c71-ae7a1dd61975.png)

**Comment by biuti** [comment]
I tried and They are correctly kept with their status.
![image](https://user-images.githubusercontent.com/35658597/114910728-873c1000-9e1e-11eb-9983-66c12c914d72.png)
![image](https://user-images.githubusercontent.com/35658597/114910802-9d49d080-9e1e-11eb-8717-ef3a52331e3f.png)

**Comment by philderbeast** [comment]
Yes I'm redoing the import and see there's a trap I fell into. I had not checked the status next to the big red [delete track] buttons. I'd assumed these were the pilots with tracks, not those that are absent. What sense does it make to [delete track] of ABS pilots?

**Comment by philderbeast** [comment]
Perhaps when ABS, the button text should reflect that? Something like, [Set NYP]?

**Comment by biuti** [comment]
Good point.
Same applies to DNF.

**Comment by biuti** [comment]
This has been updated (https://github.com/FAI-CIVL/FAI-Airscore/commit/7da4d41a2f863af2f70d0922d109cc0e7bfede73).
Let me know if it is ok.
